eZ/Publish/Core/Limitation/LanguageLimitationType.php

```diff
@@ -209,6 +212,18 @@ private function evaluateObject(ValueObject $object, APILimitationValue $value):
         return $accessVote;
     }
 
+    private function loadVersionInfo(ContentInfo $contentInfo): ?SPIVersionInfo
```

Comment:
CS/Naming: a long while ago we had an interesting naming convention. For the cases where you just tried to load something and return null if failed using exception-driven logic, you would name the method as following:
```suggestion
    private function tryLoadingVersionInfo(ContentInfo $contentInfo): ?SPIVersionInfo
```